habitat-lab/habitat/articulated_agents/robots/spot_robot.py

```diff
@@ -151,7 +156,7 @@ def _get_spot_params(self):
             leg_mtr_pos_gain=2.0,
             leg_mtr_vel_gain=1.3,
             leg_mtr_max_impulse=100.0,
-            base_offset=mn.Vector3(0.0, -0.5, 0.0),
+            base_offset=mn.Vector3(0.0, -0.48, 0.0),
```

Comment:
This will change how the Fetch robot behaves as well right? Shouldn't this be 0's by default so existing robots are not affected?